src/TextInput/TextInput.jsx

```diff
@@ -90,6 +90,7 @@ export class TextInput extends React.Component {
           required={this.props.required}
           type={type}
           value={this.props.value}
+          {...inputProps}
```

Comment:
yeah, a better pattern for passthrough props is to grab what you need from the object and pass the rest on: https://github.com/Clever/components/blob/e39cf363f6585237d03ec62e938e755efb5d2b15/src/Grid/Grid.jsx#L16

that way, the API is a little more natural:
```jsx
<TextInput className="a" myProp1="b" myProp2="c" />
```
vs
```jsx
<TextInput className="a" inputProps={{myProp1: "b" myProp2: "c"}} />
```

in some cases, we've done:
```jsx
const additionalProps = _.omit(this.props, Object.keys(TextInput.propTypes));
```
which is safer, since none of the explicit props can be overridden that way